core/src/main/java/com/netflix/conductor/core/execution/ApplicationException.java

```diff
@@ -46,11 +46,11 @@ public int getStatusCode(){
 		}
 	}
 	
-	private Code code;
+	protected Code code;
```

Comment:
Why not use `getCode()` method from L#78 instead of changing this to `protected`?

Reply:
Indeed, I think that it is not needed